pkg/packagekit/autoupdates.jsx

```diff
@@ -266,6 +266,10 @@ export class AutoUpdates extends React.Component {
                 .always(() => {
                     this.setState({ pending: false, showModal: false });
                 });
+
```

Comment:
That will also be executed when the button will be clicked, not onky if the form is submitted with onSubmit. I don't see how this would hurt us, that we preventDefault for normal footer buttons. Since they don't have some desired default behavior? So I am actually fine with it.